Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -87,6 +88,18 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	bool bExpectedActorCountsInitialised;
 	int32 ActivePlayers; // All authoritative players from all workers
 
+	// For actor migration count
+	bool bHasActorMigrationCheckFailed;
+	int32 PreviousTickMigration;
+	TArray<int32> ToBeRemovedMigrationDeltas;
+	int32 UXAuthActorCount;
+	int32 MigrationOfCurrentWorker;
+	int32 MigrationFrameCount;
+	int32 MigrationWindowFrameCount;
+	TMap<FString, int32>	MapWorkerActorMigration;
+	float MinActorMigration;
```

Comment:
nit - include unit in these types of variables

Reply:
Would you please explain it clearly for me? or give me an example like how to include a unit for MigrationFrameCount or other variables?

Reply:
ie. MinActorMigrationPerSecond or MinActorMigrationPerMillisecond